### PR TITLE
Missing `continue` statements. Better log messages

### PR DESCRIPTION
**Pull request by joeldudley**
### Why is this change needed?

While debugging another error, I realised that we don't use `continue`s properly when receiving incoming logs. Also, the log messages weren't as informative as they could have been.

### What changes were made as part of this PR:

- Provide a high level list of the changes made
- List key areas for the reviewer 

### :rotating_light: Definition of done :rotating_light:
- [ ] Unit tests added to cover new or changed functionality 
- [ ] Docs pages updated to cover new or changed functionality
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated

### CODE REVIEW

File: integration/simulation/simulation_full_network_test.go [hunk 1]
```diff
@@ -17,15 +17,15 @@ import (
...
+	numberOfNodes := 1
+	numberOfSimWallets := 1
 
 	wallets := params.NewSimWallets(numberOfSimWallets, numberOfNodes, integration.EthereumChainID, integration.ObscuroChainID)
 
 	simParams := &params.SimParams{
 		NumberOfNodes:         numberOfNodes,
 		AvgBlockDuration:      1 * time.Second,
-		SimulationTime:        45 * time.Second,
+		SimulationTime:        45 * time.Hour,
```
Comment (BedrockSquirrel) [inline integration/simulation/simulation_full_network_test.go:28]: Best not leave this in :)

File: tools/walletextension/main/cli.go [hunk 2]
```diff
@@ -18,15 +18,15 @@ const (
 	walletExtensionPortWSUsage   = "The port on which to serve websocket JSON RPC requests. Default: 3001."
 
 	nodeHostName    = "nodeHost"
-	nodeHostDefault = "testnet.obscu.ro"
+	nodeHostDefault = "localhost"
 	nodeHostUsage   = "The host on which to connect to the Obscuro node. Default: `testnet.obscu.ro`."
 
 	nodeHTTPPortName    = "nodePortHTTP"
-	nodeHTTPPortDefault = 13000
+	nodeHTTPPortDefault = 37400
```
Comment (BedrockSquirrel) [inline tools/walletextension/main/cli.go:25]: local test changes?

**Review by BedrockSquirrel — APPROVED** [review]
LGTM, test changes left in

File: tools/walletextension/main/cli.go (hunk 2 above)
Comment (joeldudley) [reply tools/walletextension/main/cli.go]: Fixed

**Pull request merged**